docs/api/changelog.rst

```diff
@@ -6,6 +6,15 @@ All notable changes to this project will be documented in this file.
 The format is based on `Keep a Changelog`_, and this project adheres to
 `Semantic Versioning`_.
 
+Unreleased
+------
+
+Added
+~~~~~
+- Added function :func:`imod.mf6.open_dvs` to read depended varibale output files like the water content file of `imod.mf6.uzf`
```

Comment:
> read depended varibale

Uhhh what is this spelling

For `imod.mf6.uzf` you want a sphinx directive, probably the class?

Reply:
done